docs/config.rst

```diff
@@ -353,13 +348,83 @@ Or, if the option needs to be available as a function parameter::
         return (conf.some_setting if val is None else val) + 2
 
 
-See Also
-========
+.. _config-affiliated:
+
+Affiliated Packages
+===================
+
+If you maintain an affiliated package you can customise the behaviour of the
+astropy configuration machinery for your package. Prior to version 4.0 of
+Astropy the bahviour of both ``astropy`` and the package template was to write a
+configuration file to ``~/.astropy/yourpackage.cfg`` if a template configuration
+file was provided by your package. If you wish to maintain this behaviour you do
+not have to change anything.
```

Comment:
I find that this paragraph directly contradicts the next one. Here, it says I don't have to change anything to retain the classic behavior, but the next paragraph says I need a wrapper to mimic what `astropy` is doing, which is also the classic behavior.

In addition, the first paragraph is also misleading, because when affiliated packages uses config system from `astropy>=4`, it does not automatically write on import anymore no matter how you configure it, you always have to run the wrapper below to generate a new file, right?